sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusRetryPolicy.cs

```diff
@@ -102,18 +102,24 @@ public abstract class ServiceBusRetryPolicy
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
 
-        /// <summary>
-        ///
-        /// </summary>
-        /// <param name="operation"></param>
-        /// <param name="scope"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        internal async Task RunOperation(
-            Func<TimeSpan, Task> operation,
+        internal async ValueTask RunOperation<T1>(
+            Func<T1, TimeSpan, CancellationToken, ValueTask> operation,
+            T1 t1,
+            TransportConnectionScope scope,
+            CancellationToken cancellationToken) =>
+            await RunOperation(static async (value, timeout, token) =>
+            {
+                var (t11, op) = value;
```

Comment:
nit:  Can we expand the naming and avoid abbreviations here?

Reply:
Any suggestions?